### ISSUE TITLE
typing dead-acute + space results in an acute accent character (U+00B4) instead of an apostrophe (U+0027)

### ISSUE THREAD

**Issue by dashea** [dashea/us-intl-android#1]

**Comment by no-mood** [comment]
Any update on this? It would be perfect

**Comment by dashea** [comment]
no. the problem is that when specifying a key layout with dead keys, it's up to android itself how to interpret a dead key followed by another keystroke, and it has some annoying decisions. fixing this would require a more intrusive program to listen to and interpret keyboard input, and I haven't had the time or inclination yet to figure out how that would even work. sorry.
